src/main/cpp/Constants.cpp

```diff
@@ -0,0 +1,42 @@
+#include "Constants.h"
+
+namespace RobotConstants {
+    // although I think getPosition() might return in native units of
+    // rotations so this would then be 1
+    const double neo550TicksPerRev = 42;
+
+    const double neoTicksPerRev = 42;
+    const double falconTicksPerRev = 2048;
+
+    // if enabled, disables a lot of the automatic shuffleboard values
+    const bool competition = false;
+} // namespace RobotConstants
+
+namespace DrivetrainConstants {
+    const double ticksPerRev = RobotConstants::neoTicksPerRev;
+    const double gearRatio = 1 / 9.64;
+    const double wheelDiameter = 6;
+
+} // namespace DrivetrainConstants
+
+namespace ShooterConstants {} // namespace ShooterConstants
```

Comment:
wait, did we actually not have any shooter constants? that's whack. I might've used the CTRE tuner tool to do all that and then just saved them in the flash memory of the Falcon 500. Not good that I didn't write them down anywhere. There might be some magic numbers in the java shooter code then that should come over here.